gpbft/participant.go

```diff
@@ -160,6 +160,7 @@ func (p *Participant) beginInstance() error {
 	if p.granite, err = newInstance(p, p.nextInstance, chain, power, beacon); err != nil {
 		return fmt.Errorf("failed creating new granite instance: %w", err)
 	}
+	p.granite.tracer = p.tracer
```

Comment:
Move to `newInstance` as constructor arg since there is a constructor for `granite`. That would limit scope for error in case `newInstrance` manipulates `tracer` as a field that's owned by `instance` struct.